lib/flows/login-flow.js

```diff
@@ -12,28 +12,26 @@ import * as dataHelper from '../data-helper';
 const host = dataHelper.getJetpackHost();
 
 export default class LoginFlow {
-	constructor( driver, account ) {
+	constructor( driver, accountOrFeatures ) {
 		this.driver = driver;
-
-		if ( account ) {
-			this.account = account;
+		accountOrFeatures = accountOrFeatures || 'defaultUser';
+		if ( typeof accountOrFeatures === 'string' ) {
```

Comment:
I don't want to break backward compatibility with existing config so we should make sure any additional code works for both types of account config.